cluster/examples/kubernetes/ceph/storageclass.yaml

```diff
@@ -20,3 +20,8 @@ parameters:
   clusterNamespace: rook-ceph
   # Specify the filesystem type of the volume. If not specified, it will use `ext4`.
   fstype: xfs
+  # (Optional) Specify an existing Ceph user that will be used for mounting storage with this StorageClass.
+  #mountUser: user1
+  # (Optional) Specify an existing Kubernetes Secret name containing just one key holding the Ceph user secret.
+  # The secret must exist in each namespace(s) where the storage will be consumed!
```

Comment:
should there also be a secret namesapce?

Reply:
@bassam No, the namespace of the Pod mounting the storage will be used to get the secret. This is in "conformance" with some existing in-tree plugins looking up the secret in the namespace of the pod using storage.

Reply:
all storage classes that have a secret also specify the namespace. why this different? https://kubernetes.io/docs/concepts/storage/storage-classes/#ceph-rbd

Reply:
That seems to be new for the user provided mount secret ..
https://v1-11.docs.kubernetes.io/docs/concepts/storage/storage-classes/#ceph-rbd
Sure, I'll add that too though would default it to pod namespace.